haskala/modules/custom/haskala_book/haskala_book.module

```diff
@@ -16,6 +16,56 @@ function haskala_book_ctools_plugin_directory($module, $plugin) {
 }
 
 /**
+ *  Translation tab - Other names for the original author.
+ *
+ *  @param $person_nid
+ *   The author person node ID.
+ *
+ *  @return  The created field for orginal author other names.
+ */
+function haskala_book_author_names($person_nid) {
```

Comment:
This function should probably be named haskala_person_get_name() or something similar, and be part of haskala_person